core/lib/zksync_core/src/house_keeper/fri_gpu_prover_archiver.rs

```diff
@@ -0,0 +1,52 @@
+use prover_dal::{Prover, ProverDal};
+use zksync_db_connection::connection_pool::ConnectionPool;
+
+use crate::{house_keeper::periodic_job::PeriodicJob, metrics::HOUSE_KEEPER_METRICS};
+
+/// FriGpuProverArchiver is a task that periodically archives old fri GPU prover records.
+/// The task will archive the `dead` prover records that have not been updated for a certain amount of time.
```

Comment:
This comment does not correspond to the logic defined in `dal`

Reply:
That is actually mistake of the dal implementation, thanks for noticing!

Reply:
Should be okay now